src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -127,7 +125,8 @@ public void Initialize(StorageEnvironment environment, TransactionContextPool co
                 _licenseStatus.FirstServerStartDate = firstServerStartDate.Value;
 
                 ReloadLicense(firstRun: true);
-                AsyncHelpers.RunSync(() => CalculateLicenseLimits());
+
+                AsyncHelpers.RunSync(() => _serverStore.PutMyNodeInfoAsync(_licenseStatus.MaxCores));
```

Comment:
This may wait (a *lot*) if the cluster isn't setup.
We are waiting for this in a sync manner, and it can hold up the server startup, no?

Reply:
Done